source/features/patch-diff-links.tsx

```diff
@@ -14,19 +14,23 @@ async function addPatchDiffLinks(commitMeta: HTMLElement): Promise<void> {
 	}
 
 	commitMeta!.classList.remove('no-wrap'); // #5987
-	commitMeta!.prepend(
-		<span className="sha-block" data-turbo="false">
-			<a href={`${commitUrl}.patch`} className="sha color-fg-default">patch</a>
-			{' '}
-			<a href={`${commitUrl}.diff`} className="sha color-fg-default">diff</a>
-		</span>,
-		<span className="px-2">·</span>,
+	// TODO: Improve after April 2025
+	const node = (
+		<>
+			<span className="sha-block" data-turbo="false">
+				<a href={`${commitUrl}.patch`} className="sha color-fg-default">patch</a>
+				{' '}
+				<a href={`${commitUrl}.diff`} className="sha color-fg-default">diff</a>
+			</span>
+			{commitMeta.tagName !== 'DIV' && <span className="px-2">·</span>}
```

Comment:
You can use a ternary operator with `=== DIV ? undefined : <span>` instead of using <>